deploy/operator-role-olm.yaml

```diff
@@ -47,6 +47,15 @@ rules:
   - patch
   - update
   - watch
+- apiGroups:
+  - "policy"
+  resources:
+  - poddisruptionbudgets
+  verbs:
+  - create
+  - delete
+  - get
+  - update
```

Comment:
`patch` verb is probably to be added.

Also, `operator-role-kubernetes.yaml` must be updated.

Reply:
actually `list` is missing too..